include/circt/Dialect/MSFT/MSFTOps.td

```diff
@@ -198,7 +191,10 @@ def MSFTModuleOp : MSFTModuleOpBase<"module",
 }
 
 def MSFTModuleExternOp : MSFTOp<"module.extern",
-      [Symbol, FunctionOpInterface, HasParent<"mlir::ModuleOp">, DeclareOpInterfaceMethods<HWModuleLike>]> {
+      [Symbol, FunctionOpInterface, HasParent<"mlir::ModuleOp">, 
+      DeclareOpInterfaceMethods<PortList>,
+      DeclareOpInterfaceMethods<SymboledPortList>,
```

Comment:
Does this dialect actually use port symbols? (and similarly the others, like ESI)?

I think they may only stub that out because they couldn't opt-in to the features they wanted -- presently looks like they uncondtionally return empty attributes?  Maybe now's the opportunity to drop that until it's useful/meaningful?

Reply:
Yea, they use a HWMutableModule, but don't implement most of hwmodulelike, but it's really painful to change.  Changing firrtl was easier than those dialects (hence this PR :) )